hs-bindgen/hs-bindgen.cabal

```diff
@@ -321,7 +321,7 @@ test-suite test-th
 
 test-suite test-pp
   type:           exitcode-stdio-1.0
-  main-is:        test-pp.hs
+  main-is:        ../th/test-th.hs
```

Comment:
> The workaround in the cabal file is perhaps a bit unsavoury but it seems to work.

Add a comment or I'll reintroduce the symlink once I forget why there is a HACK.